torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -675,17 +679,18 @@ class ExecutorchCallDelegateHigherOrderVariable(TorchHigherOrderOperatorVariable
     def call_function(
         self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
     ) -> "VariableTracker":
-        from . import ConstantVariable
         from .builder import wrap_fx_proxy
 
-        self.check_kwargs(kwargs, ConstantVariable)
-
         # This is operator for delegation within Executorch which calls a
         # specific function in the given lowered module with the given
         # operators. The actual operator is defined in the Executorch codebase.
         # This is a bad hierarchical violation since
         # executorch_call_delegate sits at a higher level than dynamo, but
         # there's no real solution to this issue yet.
+        if len(kwargs) > 0:
+            unimplemented(
+                "executorch_call_delegate: kwargs arguments were not enabled."
+            )
```

Comment:
@tugsbayasgalan, @angelayi does this look right?

Reply:
The original PR introducing these args were the migration by @anijain2305

Reply:
@anijain2305 would you care to comment? I don't think your changes were warranted as the kwargs were never consumed. Hence let's remove them.

As we can see, all of the other higher order ops gradually removed those changes. So these are the last to go.

Reply:
Animesh is out of office. But anyways, I think the changes in this PR look good, as long as CI passes.